scripts/checkdeps

```diff
@@ -20,32 +20,26 @@
 
 . config/options $1
 
-deps=
-deps_pkg=
-files=
-files_pkg=
-
-case $1 in
-  get)
-    deps="wget"
-    deps_pkg="wget"
-    ;;
-  build)
-    deps="bash bc gcc g++ sed patch touch tar bzip2 gzip perl cp gawk makeinfo gperf zip unzip mkfontscale mkfontdir bdftopcf diff xsltproc java md5deep"
-    deps_pkg="bash bc gcc g++ sed patch fileutils tar bzip2 gzip perl coreutils gawk texinfo gperf zip unzip xfonts-utils xfonts-utils xfonts-utils diff xsltproc default-jre md5deep"
-    files="/usr/include/stdio.h /usr/include/ncurses.h"
-    files_pkg="libc6-dev libncurses5-dev"
-    if [ -f /etc/redhat-release ]; then
-      deps_pkg="bash bc gcc gcc-c++ sed patch fileutils tar bzip2 gzip perl coreutils gawk texinfo gperf zip unzip xorg-x11-font-utils xorg-x11-font-utils diffutils libxslt java-1.7.0-openjdk"
+if  [ -f /etc/lsb-release ]; then
+  DISTRO=$(grep DISTRIB_ID /etc/lsb-release | cut -d "=" -f 2 | tr A-Z a-z)
+fi
+
+if  [ -f /etc/os-release ]; then
+  DISTRO=$(grep ^ID= /etc/os-release | cut -d "=" -f 2 | tr A-Z a-z)
+fi
+
+deps="wget bash bc gcc g++ sed patch touch tar bzip2 gzip perl cp gawk makeinfo gperf zip unzip mkfontscale mkfontdir bdftopcf diff xsltproc java md5deep"
+deps_pkg="wget bash bc gcc g++ sed patch fileutils tar bzip2 gzip perl coreutils gawk texinfo gperf zip unzip xfonts-utils xfonts-utils xfonts-utils diff xsltproc default-jre md5deep"
+files="/usr/include/stdio.h /usr/include/ncurses.h"
+files_pkg="libc6-dev libncurses5-dev"
+
+case "$DISTRO" in
+    fedora|centos|rhel)
+      deps_pkg="wget bash bc gcc gcc-c++ sed patch fileutils tar bzip2 gzip perl coreutils gawk texinfo gperf zip unzip xorg-x11-font-utils xorg-x11-font-utils diffutils libxslt java-1.7.0-openjdk"
```

Comment:
We talked about this before I think and we need glibc-static and libstdc++-static on fedora